packages/myst-to-react/src/basic.tsx

```diff
@@ -363,6 +368,10 @@ const BASIC_RENDERERS: BasicNodeRenderers = {
       </dd>
     );
   },
+  keyboard({ node }) {
+    const valueNode = (node.children as GenericNode[])[0];
+    return <kbd>{valueNode.value}</kbd>;
```

Comment:
I think we should use `<Myst children>` explicitly, which does that, but won't break if it isn't the case.